app/controllers/hiring_staff/base_controller.rb

```diff
@@ -1,32 +1,43 @@
 class HiringStaff::BaseController < ApplicationController
   before_action :authenticate
+  DEFAULT_SCHOOL_URN = '140181'.freeze
 
   def authenticate
     return unless authenticate_hiring_staff?
     authenticate_or_request_with_http_basic('Hiring Staff') do |name, password|
-      name == http_user && password == http_pass
+      if name == benwick_http_user && password == benwick_http_pass
+        session[:urn] = '110627'
+        true
+      elsif name == http_user && password == http_pass
+        session[:urn] = DEFAULT_SCHOOL_URN # Enables the product team to access publishing and the test suite a default
+        true
+      end
     end
   end
 
+  def current_school
+    @current_school ||= School.find_by! urn: session[:urn]
+  rescue ActiveRecord::RecordNotFound
+    not_found
+  end
+
   def authenticate_hiring_staff?
-    !(Rails.env.development? || Rails.env.test?)
+    true
```

Comment:
Is there any value in having `authenticate_hiring_staff?` anymore then or was this removed accidentaly?

Reply:
Oh good spot. This can go :)